syncstorage/db.go

```diff
@@ -217,23 +219,22 @@ func NewDB(path string, conf *Config) (*DB, error) {
   work is handled by private functions.
 */
 
-// LastModified gets the database modified time
-func (d *DB) LastModified() (modified int, err error) {
+// LastModified returns the top level last modified timestamp
+func (d *DB) LastModified() (int, error) {
 	d.Lock()
 	defer d.Unlock()
 
-	var m sql.NullInt64
+	lastMod, err := getKey(d.db, STORAGE_LAST_MODIFIED)
+	if lastMod == "" || err != nil {
+		return 0, err
```

Comment:
If the value doesn't exists it will return 0, maybe we can scrape the max value to initialize it. and in case of error we will return 0